goreleaser-nightly.yaml

```diff
@@ -0,0 +1,73 @@
+project_name: gh
```

Comment:
Not strictly necessary, but I got into playing with `goreleaser` and this will allow publishing nightly builds to the edge channel.